pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -126,7 +126,19 @@ func GetDomainSpecWithRuntimeInfo(dom cli.VirDomain) (*api.DomainSpec, error) {
 		return nil, err
 	}
 
-	metadataXML, err := dom.GetMetadata(libvirt.DOMAIN_METADATA_ELEMENT, "http://kubevirt.io", libvirt.DOMAIN_AFFECT_CONFIG)
+	// use different flag with GetMetadata for transient domains
```

Comment:
lets separate this logic into it's own function, maybe `getDomainImpactFlag` or something?